app/views/groups/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div id="expenditures">
+<div class="header-div">
+   <h1 class="header"> Transactions </h1>
+</div>
+<h1> <%= link_to user_groups_path do %>
+ <i class="fa-solid fa-chevron-left" style="color: #3778c2; padding-left: 22px"> </i>
+ <% end %>
+ </h1>
+ <div class="all-expense">
+ <div class="total"> <span class="expense">Total </span>
+ <span> <%= number_to_currency(@expenditures.sum(:amount)) %> </span>
+ </div>
+  <% @expenditures.each do |expenditure| %>
+  <%= render expenditure %>
+  <% end %>
+    <div class="new-form"> 
+   <span> <%= link_to 'New Expenditure', new_user_expenditure_path %> </span>
+  </div>
+  </div>
+</div>
```

Comment:
- [ ] When the user clicks on the "Back" button (<), the user navigates to the home page